conf/locale/ar/LC_MESSAGES/django.po

```diff
@@ -10445,7 +10445,7 @@ msgid "Confirm Email"
 msgstr ""
 
 #: openedx/core/djangoapps/user_api/views.py
-msgid "Please confirm your email."
+msgid "The email addresses do not match."
```

Comment:
the .po files are generated from some automated process, so they shouldn't be changed manually here.